### ISSUE TITLE
Are images sorted in the classes?

### ISSUE THREAD

**Issue by dsmic** [yaoyao-liu/mini-imagenet-tools#4]
I use the files from the "Download Processed Images" (the tar file).

To me it seems that the images become sorted using MiniImageNetDataLoader, and the first images for each label are simpler to classify than later ones? 

If I use `shuffle = True` at the end of the middle line (there are three occurrences of such lines)

```
  random.shuffle(sampled_character_folders)
  labels_and_images = self.get_images(sampled_character_folders, range(self.way_num), nb_samples=self.num_samples_per_class, shuffle=False)
  labels = [li[0] for li in labels_and_images]
``` 
it is much more difficult to train the data.

I got suspicious because the default in
```
def get_images(self, paths, labels, nb_samples=None, shuffle=True):
```
which is overwritten in the call.

**Comment by yaoyao-liu** [comment]
Thanks for your interest in this project.

If you use the default libraries of TensorFlow or PyTorch to build the deep neural networks, the order of samples in a single batch will be ignored. So the default setting for `shuffle` is `False`. You could set it `True`. It depends on your code.

If you hope to see an example of how this repo work in a real case, you may see: [Meta-Transfer Learning](https://github.com/y2l/meta-transfer-learning-tensorflow).

**Comment by dsmic** [comment]
Thanks a lot for the very fast reply. I had an ugly error in my code which lead this assumption...

**Comment by yaoyao-liu** [comment]
Welcome. Feel free to create another issue if you have other questions.
